### PR TITLE
Implement Shell in `CommunityToolkit.Maui.Markup.Sample`

### PR DESCRIPTION
**Pull request by brminnick**
### Description of Change ###

This PR updates `CommunityToolkit.Maui.Markup.Sample` to use Shell

### CODE REVIEW

File: samples/CommunityToolkit.Maui.Markup.Sample/AppShell.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	public AppShell(NewsPage newsPage)
+	{
+		Items.Add(new ShellContent
+		{
+			Title = newsPage.Title,
+			Content = newsPage
+		});
+
+		//  BarBackgroundColor = ColorConstants.NavigationBarBackgroundColor;
+		//	BarTextColor = ColorConstants.NavigationBarTextColor
```
Comment (pictos) [inline samples/CommunityToolkit.Maui.Markup.Sample/AppShell.cs:29]: Do we need these comments here?

**Review by pictos — COMMENTED** [review]
Everything looks good to me, I found a small detail.

File: samples/CommunityToolkit.Maui.Markup.Sample/AppShell.cs (hunk 1 above)
Comment (brminnick) [reply samples/CommunityToolkit.Maui.Markup.Sample/AppShell.cs:29]: Good catch! We sure don't!

I forgot to remove those after my debugging.

File: samples/CommunityToolkit.Maui.Markup.Sample/AppShell.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+	public AppShell(NewsPage newsPage)
+	{
+		Items.Add(newsPage);
+	}
+
+	public static string GetRoute<TPage, TViewModel>() where TPage : BaseContentPage<TViewModel>
+														where TViewModel : BaseViewModel
+	{
+		if (!pageRouteMappingDictionary.ContainsKey(typeof(TPage)))
```
Comment (pictos) [inline samples/CommunityToolkit.Maui.Markup.Sample/AppShell.cs:25]: Can we use the `TryGet` method? That way we will do the _Key verification_ and grab the value in one move.

**Review by pictos — DISMISSED** [review]
I've one suggestion, but that doesn't block this PR! If you think that doesn't makes sense feel free to merge (:

File: samples/CommunityToolkit.Maui.Markup.Sample/AppShell.cs (hunk 2 above)
Comment (brminnick) [reply samples/CommunityToolkit.Maui.Markup.Sample/AppShell.cs]: Great idea! Done ✅

**Review by pictos — APPROVED** [review]

**Pull request merged**